vpr/src/base/vpr_types.h

```diff
@@ -1076,6 +1076,8 @@ struct t_placer_opts {
     float place_crit_limit;
     int place_constraint_expand;
     bool place_constraint_subtile;
+    int floorplan_num_horizontal_partitions;
```

Comment:
Placer opts doesn't seem like the right place for this. Seems like a constraints opts structure would be worthwhile.